template-parts/header_content-archive.php

```diff
@@ -0,0 +1,26 @@
+<?php
+/**
+ * Header content template for the 'Section' page template
+ */
+?>
+
+<?php
+global $post;
```

Comment:
`global $post;` can be removed--it's not referenced anywhere, and I don't think it'll actually be set in archive views anyway

Reply:
Removed.